Add script to quickly generate a machine info template

punchagan opened the pull request:
The script will need to be run on the machine whose information we need

mtelvers commented:
Thanks Puneeth.  I've been meaning to add a script for some time now.

We additionally need to collect memory size and disks (which is a bulleted list) (lsblk).

I noticed that you have added some extra fields such as use, service and notes. This is currently handled with just free form text at after the yaml header. If we really want to do this, which I'm not convinced that we do, we need to make corresponding changes to the page generation code.

Apologies for the short message and typos as I'm on my mobile!

punchagan commented:
> Thanks Puneeth. I've been meaning to add a script for some time now.

Updated the script to include memory and disks information. 

Also, added an additional commit to include this info in pascal's file.

shonfeder commented:
Anything preventing us from merging this?
